File: TwoSum/c++/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ret;
        int len = nums.size();
        for(int i = 0; i < len-1; i++){
            for(int j = i+1; j < len; j++){
                if(nums[i]+nums[j]==target){
                    ret.push_back(i);
                    ret.push_back(j);
                    
                    return ret;
                }
            }
        }
        return ret;
    }
};

File: TwoSum/C#/TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int len = nums.Length;
        for(int i = 0; i< len-1; i++){
            for(int j = i+1; j<len; j++){
                if(nums[i]+nums[j]==target){
                    return new int[] {i,j};
                }
            }
        }
        return null;
    }
}
